### ISSUE TITLE
Many icon buttons need alt text

### ISSUE THREAD

**Issue by shortwavesurfer2009** [aeharding/voyager#743]
![Screenshot_20230926-130928_Voyager](https://github.com/aeharding/voyager/assets/116814522/4913af72-b04c-4fbb-8ce3-924b5a15f64f)
![Screenshot_20230926-130933_Voyager](https://github.com/aeharding/voyager/assets/116814522/41b45a4d-284c-40ec-9a8e-3a4236b6e638)
![Screenshot_20230926-130942_Voyager](https://github.com/aeharding/voyager/assets/116814522/fad21fa8-0bed-4154-884c-30cf26f63baf)
![Screenshot_20230926-130945_Voyager](https://github.com/aeharding/voyager/assets/116814522/e1e65bee-2d76-44b8-ac07-64dcd8ce2aa8)
![Screenshot_20230926-130949_Voyager](https://github.com/aeharding/voyager/assets/116814522/41b3153a-f3c4-46a3-a55d-03d49541e6ef)
![Screenshot_20230926-131013_Voyager](https://github.com/aeharding/voyager/assets/116814522/1f0aa72e-6f99-403a-a858-3479bc19c0c4)
![Screenshot_20230926-131015_Voyager](https://github.com/aeharding/voyager/assets/116814522/3d13a867-da82-4d3d-a7a0-4192428306a4)
![Screenshot_20230926-131017_Voyager](https://github.com/aeharding/voyager/assets/116814522/73436a60-59cb-4509-9d85-ceddc0a132e4)
![Screenshot_20230926-131018_Voyager](https://github.com/aeharding/voyager/assets/116814522/2b337df0-0854-4549-aa42-4d258ad36adb)
![Screenshot_20230926-131020_Voyager](https://github.com/aeharding/voyager/assets/116814522/ffec490e-e3b5-4b7b-ba71-c3b335d55e51)
![Screenshot_20230926-131025_Voyager](https://github.com/aeharding/voyager/assets/116814522/9dfebbfc-bfe3-4e0f-a543-56f26840837a)
![Screenshot_20230926-131044_Voyager](https://github.com/aeharding/voyager/assets/116814522/225291b1-8cc5-4a5c-8ba0-57805e8d295a)
![Screenshot_20230926-131048_Voyager](https://github.com/aeharding/voyager/assets/116814522/df38f306-f732-4d1b-8be1-5e50a7931252)

**Comment by 80avin** [comment]
In addition to alt text, all clickable elements which are not `button` or `a`, etc and are made clickable by just onClick listener need to have proper role also.

Example is the like icon/counter is clickable, but not a `button` or doesn't have `role="button"`.

This affects the accessibility software, and for me particularly, affects the usage of [vimium](https://chromewebstore.google.com/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb) extension.
